### ISSUE TITLE
Improve usability of Filter

### ISSUE THREAD

**Issue by PhyxionNL** [exceptionless/Exceptionless.UI#201]
Right now the button at the right side of the Filter bar only has one feature: Search Documentation.
I think this can be improved with a drop down of commonly used filters, such as Fixed, Ignored, All (*) and then a Search Documentation button. Clicking on a commonly used filter should add this entry to the Filter bar.

**Comment by niemyjski** [comment]
We wanted to have the ability to save complete dashboards along with search criteria but this may be a really good intermediate step. Have you seen the new reports section? It has views with these filters?

**Comment by PhyxionNL** [comment]
Yeah, I've seen them, but it's not really the same. It would be easier and more useful to have these in the Filter options so you can also use them on other views such as Feature Usages.

**Comment by niemyjski** [comment]
Agreed
